ksqldb-tools/src/main/java/io/confluent/ksql/tools/migrations/MigrationConfig.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2021 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"; you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.tools.migrations;
+
+import java.util.Map;
+
+public class MigrationConfig  {
+
+  public static final String KSQL_MIGRATIONS_STREAM_NAME = "ksql.migrations.stream.name";
+  public static final String KSQL_MIGRATIONS_TABLE_NAME = "ksql.migrations.table.name";
+  public static final String KSQL_MIGRATIONS_STREAM_TOPIC_NAME = "ksql.migrations.stream.topic.name";
+  public static final String KSQL_MIGRATIONS_TABLE_TOPIC_NAME = "ksql.migrations.table.topic.name";
+  public static final String KSQL_MIGRATIONS_TOPIC_REPLICAS = "ksql.migrations.topic.replicas";
+  public static final String KSQL_SERVER_URL = "ksql.server.url";
+
+  private static Map<String, String> configs;
+
+  public MigrationConfig(final Map<String, String> configs) {
+    this.configs = configs;
+  }
+
+  public String getKsqlMigrationsStreamName() {
```

Comment:
Being `MigrationsConfig` an immutable object, and that only returns configs, I think we could rename the getter methods for simple names, like:
```
config.streamName();
config.streamTopic();
config.topicReplicas();
config.tableName();
config.tableTopic();
```

What do you think?

Reply:
Extended `MigrationConfig` from `AbstractConfig` instead, so all these getter functions are removed.